app/scripts/scripts.js

```diff
@@ -1,17 +1,28 @@
-/* For picking letters from alphabet keypad. Returns the
-   letter chosen or "already picked"
-*/
-function pickLetter(letter) {
+(function () {
```

Comment:
So I end up copying index.html, styles.css, and scripts.js into a separate folder and it seems to work there :) which is why I pushed my updates.